frontend/src/components/pages/admin/CreateAssessmentPage.tsx

```diff
@@ -49,17 +49,13 @@ const CreateAssessmentPage = (): React.ReactElement => {
     }
   }, [questions, errorMessage]);
 
-  const onSave: SubmitHandler<TestRequest> = async (data) => {
+  const onCreateTest = async (test: TestRequest) => {
```

Comment:
should we just have `onCreateTest` take in a status field? then we don't need `onSave` and `onPublish`

Reply:
i think it doesn't work b/c the `SubmitHandler` can't take in multiple parameters? cuz you have to pass to `handleSubmit` in the provided function signature format